e2etest/zt_newe2e_s2s_test.go

```diff
@@ -619,3 +620,39 @@ func (s *S2STestSuite) Scenario_S2SDirectoryMultipleFilesStripTopDirNonRecursive
 		Objects: dstObjs,
 	}, true)
 }
+
+func (s *S2STestSuite) Scenario_SystemContainerCopy(svm *ScenarioVariationManager) {
+	azCopyVerb := ResolveVariation(svm, []AzCopyVerb{AzCopyVerbCopy, AzCopyVerbSync}) // Calculate verb early to create the destination object early
+
+	dstObj := GetAccount(svm, PrimaryStandardAcct).GetService(svm, common.ELocation.Blob()).GetContainer("$logs")
+
+	body := NewRandomObjectContentContainer(SizeFromString("10K"))
```

Comment:
no need to actually make data for this test since its going to fail anyway. ok if we do this after the release.